### PR TITLE
Export single thought without bullet

### LINKED ISSUE #1028: Export single thought without bullet
## Steps to Reproduce

Create a single thought `a` and export it. 

## Current Behavior

Exports `- a`

## Expected Behavior

Should export `a`

This is a special case for exporting a single thought.

### PR DESCRIPTION
**Pull request by shresthabijay**
fixes #1028

### CODE REVIEW

**Comment by shresthabijay** [comment]
@raineorshine Please review the changes. Thanks!

File: src/selectors/exportContext.ts [hunk 1]
```diff
@@ -25,8 +25,7 @@ interface Options {
  * @param format
  * @param title     Replace the value of the root thought with a new title.
  */
-export const exportContext = (state: State, context: Context, format: MimeType = 'text/html', { indent = 0, title, excludeSrc, excludeMeta }: Options = {}): string => {
-  const linePrefix = format === 'text/html' ? '<li>' : '- '
+export const exportContext = (state: State, context: Context, format: MimeType = 'text/html', { indent = 0, title, excludeSrc, excludeMeta }: Options = {}, depth = 0): string => {
```
Comment (raineorshine) [inline src/selectors/exportContext.ts:28]: Can you add `depth` to the `Options` object itself? I think that's cleaner.

**Review by raineorshine — CHANGES_REQUESTED** [review]

File: src/selectors/exportContext.ts (hunk 1 above)
Comment (shresthabijay) [reply src/selectors/exportContext.ts:28]: Sure.

**Comment by shresthabijay** [comment]
@raineorshine  I have made the requested changes.

**Review by raineorshine — APPROVED** [review]

**Pull request merged**